holoviews/plotting/mpl/__init__.py

```diff
@@ -219,6 +220,14 @@ def grid_selector(grid):
 register_cmap("fire", cmap=fire_cmap)
 register_cmap("fire_r", cmap=fire_r_cmap)
 
+def mpl_cm(name,colorlist):
+    cm = LinearSegmentedColormap.from_list(name, colorlist, N=len(colorlist))
+    register_cmap(name, cmap=cm)
+
+mpl_cm('kbc_r',list(reversed(kbc)))
```

Comment:
Not sure if there's a reason to define mpl_cm here; is it ever reused?

Reply:
Seems like this should just follow the pattern that fire_cmap and fire_r_cmap follow above, or vice versa.

Reply:
Addressed in c72f83b